### PR TITLE
Added rule for profiseller.de.

### PR DESCRIPTION
**Pull request by Jochen-A-Fuerbacher**

### CODE REVIEW

**Comment by gloomy-ghost** [comment]
Thanks. Please add a comment to the top saying that the other [subdomains](https://gist.github.com/gloomy-ghost/514573c7cbefffd92acef8611b1b62d2) don't have valid certificates, and most of them are just redirecting to the main site.

Rewrite `p[\d]+.profiseller.de` and `www.p[\d]+.profiseller.de` to `https://www.profiseller.de/shop1/index.php3?ps_id=%number%` if you want.

**Comment by Jochen-A-Fuerbacher** [comment]
@gloomy-ghost Done.

**Comment by gloomy-ghost** [comment]
The list is too long so please don't list them. You can just use the suggestion at https://github.com/EFForg/https-everywhere/issues/6863#issuecomment-260018136

Also the rules need `test`s, see https://travis-ci.org/EFForg/https-everywhere/jobs/180930047#L211

File: src/chrome/content/rules/profiseller.de.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<ruleset name="Profiseller">
+
+        <!-- Some subdomains don't have valid certificates. -->
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/profiseller.de.xml]: And redirect to `www`

**Comment by Jochen-A-Fuerbacher** [comment]
@gloomy-ghost Thanks for your feedback. Added tests.

**Comment by gloomy-ghost** [comment]
Also `rule`s won't take effects unless the targets are listed.  I suggest a wildcard here. (Note: rewrite `^http://(www\.)profiseller\.de/` and remove `^http:` after add a wildcard target)

**Comment by gloomy-ghost** [comment]
Sorry for small pieces of comments, I'm typing on my phone :-|

**Comment by gloomy-ghost** [comment]
Thanks, merged.

**Pull request merged**